package.json

```diff
@@ -14,7 +14,7 @@
         "build:internal": "cross-env USE_JSON_DATASET=true webpack --config ./webpack/webpack.config.js  --env env=development",
         "start": "webpack serve --config ./webpack/webpack.config.js --env env=development",
         "start:internal": "cross-env USE_JSON_DATASET=true webpack-dev-server --config ./webpack/webpack.config.js --env env=development",
-        "start:dev-db": "cross-env USE_DEV_DB=true webpack serve --config ./webpack/webpack.config.js  --env env=development",
+        "start:dev": "cross-env USE_DEV_DB=true USE_REDUX_DEVTOOLS=true webpack serve --config ./webpack/webpack.config.js  --env env=development",
```

Comment:
This is a good approach but I am not sure we want to mix redux devtools with dev-db.  I'm thinking we need to be able to independently point the app to dev database vs production database, and separately make the decision to run with or without redux-devtools. 

I'm ok with waiting till Megan gets back to have her weigh in here.  Can we get extra command line arguments into `npm run` scripts? That would solve this too.

Reply:
I haven't tried this but I think you can do 
`npm run scriptName --argName=argVal`
and then access the argName to get its value:
`process.env.npm_config_argName`

You can do multiple args via:
`npm run scriptName -- arg1=val1 arg2=val2`
or
`npm run scriptName --arg1=val1 --arg2=val2`